app/javascript/components/admin/UserModal.js

```diff
@@ -20,6 +21,11 @@ class UserModal extends React.Component {
     this.setState({ email: val });
   };
 
+  handleNotesChange = e => {
```

Comment:
this is the same function as the `handleEmailChange`. can you combine into one `handleChange` event for both like we do elsewhere?